packages/react/src/components/tab/Tab.tsx

```diff
@@ -82,6 +81,8 @@ export const Tab: FunctionComponent<ITabProps> = ({
         return onDestroy;
     }, []);
 
+    const IconName = icon;
```

Comment:
Wouldn't it be better to directly rename the props when destructuring?

```
export const Tab: FunctionComponent<ITabProps> = ({
    icon: IconName,
```

Reply:
When what? xD